bwapi/authenticate.py

```diff
@@ -33,13 +32,24 @@ def main():
 
     parser = argparse.ArgumentParser(
         description="Logging to Brandwatch and retrieve and access token.",
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
 
-    parser.add_argument("--store", "-s", type=Path, metavar="PATH", default=credentials.DEFAULT_CREDENTIALS_PATH,
-                        help="Path to where access tokens are stored.")
+    parser.add_argument(
+        "--store",
+        "-s",
+        type=Path,
+        metavar="PATH",
+        default=credentials.DEFAULT_CREDENTIALS_PATH,
+        help="Path to where access tokens are stored.",
+    )
```

Comment:
Not a big deal but it would be slightly easier to review this PR if each type of change was in a separate commit. For example, there's a lot of whitespace changes that could be ignored, but I have to read everything because they mixed in with other stuff.

Reply:
Yup.  I should have done them separately.  Happy to discard the PR and resubmit as 2 separate PRs for understandability.

Reply:
No don't worry about it. I was just a "something to bear in mind of next time" sort of thing :)